src/javascripts/helpers/validationForm.js

```diff
@@ -0,0 +1,17 @@
+import { REGEX } from "../constants/config";
+
+
+export const isRequired = (data) => {
+    for (const key in data) {
```

Comment:
It's good, but you should consider "Array.find"

Reply:
@phuocduong-agilityio  Because I use a JavaScipt object as the passed parameter like this ( image ). So I think using Array.find here is not reasonable
![image](https://github.com/HoangLiarPhuongLe/javascript__training/assets/110210274/48072c9c-2e08-4c09-97d9-6ebb465b3550)